specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingAccount.json

```diff
@@ -70,960 +119,750 @@
+          },
+          {
+            "$ref": "./types.json#/parameters/orderBy"
+          },
+          {
+            "$ref": "./types.json#/parameters/top"
+          },
+          {
+            "$ref": "./types.json#/parameters/skip"
+          },
+          {
+            "$ref": "./types.json#/parameters/count"
+          },
+          {
+            "$ref": "./types.json#/parameters/search"
           }
         ],
         "responses": {
           "200": {
-            "description": "OK. The request has succeeded.",
+            "description": "The list of invoice section properties with create subscription permission.",
             "schema": {
-              "$ref": "#/definitions/ValidateAddressResponse"
+              "$ref": "#/definitions/InvoiceSectionWithCreateSubscriptionPermissionListResult"
             }
           },
           "default": {
-            "description": "Error response describing why the operation failed.",
+            "description": "Default response",
             "schema": {
-              "$ref": "./types.json#/definitions/ArmError"
+              "$ref": "./types.json#/definitions/ArmErrorResponse"
             }
           }
         },
-        "tags": [
-          "Address"
-        ],
         "x-ms-examples": {
-          "addressValidateInvalid": {
-            "$ref": "./examples/addressValidateInvalid.json"
-          },
-          "addressValidateValid": {
-            "$ref": "./examples/addressValidateValid.json"
+          "InvoiceSectionsListWithCreateSubPermission": {
+            "$ref": "./examples/invoiceSectionsListWithCreateSubPermission.json"
           }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
         }
       }
     }
   },
   "definitions": {
-    "AddressValidationStatus": {
-      "description": "Status of the address validation.",
-      "enum": [
-        "Valid",
-        "Invalid"
-      ],
-      "type": "string",
-      "x-ms-enum": {
-        "modelAsString": true,
-        "name": "addressValidationStatus"
-      }
-    },
     "BillingAccount": {
+      "description": "A billing account.",
+      "type": "object",
       "allOf": [
         {
           "$ref": "./types.json#/definitions/ArmResource"
         }
       ],
-      "description": "A billing account.",
       "properties": {
         "properties": {
-          "$ref": "#/definitions/BillingAccountProperties",
-          "description": "The properties of the billing account.",
+          "description": "The properties of a(n) BillingAccount",
+          "allOf": [
+            {
+              "$ref": "#/definitions/BillingAccountProperties"
+            }
+          ],
           "x-ms-client-flatten": true
         }
       },
-      "type": "object"
+      "x-ms-azure-resource": true
     },
     "BillingAccountListResult": {
-      "description": "The list of billing accounts.",
+      "description": "A container for a list of resources",
```

Comment:
In some places we're saying "list of resources" while in other places we're saying "list of Billing Accounts"... Can we make it consistent?